iib/web/api_v1.py

```diff
@@ -90,6 +91,23 @@ def get_builds():
     return flask.jsonify(response)
 
 
+@api_v1.route('/healthcheck')
+def get_healthcheck():
+    """
+    Respond to a health check.
+
+    :rtype: str
+    """
+    # Test DB connection
+    try:
+        db.engine.execute(text('SELECT 1'))
+    except Exception as e:
+        flask.current_app.logger.error('DB test failed with the following exception: %s', str(e))
```

Comment:
If the exception details were not logged appropriately before this change, they're still not logged appropriately - str(e) is throwing away the backtrace entirely.  str(e) alone could be something virtually useless like "KeyError: foo".

If you did logger.exception("DB test failed"), or logger.error("DB test failed", exc_info=True), it would log the exception including backtrace.  I think this is the correct way to log exceptions.

Reply:
Fixed.